dev.README

```diff
@@ -0,0 +1,16 @@
+```
+make shell
+ln -s mobile_files/package.json.orig package.json
+yarn install
+shadow-cljs watch android
```

Comment:
Maybe instead of making a new file we shoule update/expand [`STARTING_GUIDE.md`](https://github.com/status-im/status-react/blob/develop/STARTING_GUIDE.md)?

Reply:
actually this file is useless I forgot to remove it